openff/evaluator/properties/properties.py

```diff
@@ -192,6 +196,224 @@ def default_equilibration_schema(
         )
         return calculation_schema
 
+    @classmethod
+    def default_preequilibrated_simulation_schema(
```

Comment:
I tried to reason through how some of this partially-duplicated code could be consolidated down, but didn't get anywhere

Reply:
I think just passing in the input schema object and generating protocols from there would be an improvement.

There are other things that'd be great to fix, like how hard it is right now to change details like the simulation runtime from the schema that's output from this method... but things that can also happen in the future.